Modules/ThirdParty/GoogleTest/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ if(NOT ITK_BINARY_DIR)
   set(GTEST_ROOT \"${GTEST_ROOT}\")
   find_package(GTest REQUIRED)
 endif()
+")
+    set(ITKGoogleTest_EXPORT_CODE_INSTALL "
+  set(GTEST_ROOT \"${GTEST_ROOT}\")
```

Comment:
The install tree is relocatable so we shouldn't set `GTEST_ROOT` since GTest might not be in the same place after deployment.